src/osd/OSDMap.cc

```diff
@@ -3965,10 +3965,13 @@ void OSDMap::print_summary(Formatter *f, ostream& out,
 			   const string& prefix, bool extra) const
 {
   if (f) {
+    utime_t now = ceph_clock_now();
     f->dump_int("epoch", get_epoch());
     f->dump_int("num_osds", get_num_osds());
     f->dump_int("num_up_osds", get_num_up_osds());
+    f->dump_int("osd_up_since", (now - last_up_change).to_msec() / 1000);
```

Comment:
Is output of some large number of seconds really that useful?

Reply:
> Is output of some large number of seconds really that useful?

@socketpair I also think the number is very large. Could I change it back to utimespan_str?

Reply:
nope, JSON is a machine-centric output. number of seconds is the most convenient (useful) information. Human readable datetime requires parsing.